### PR TITLE
reStructuredText Citations

### PR DESCRIPTION
**Pull request by atavory**
Using reStructuredText's [dedicated directive](http://docutils.sourceforge.net/docs/user/rst/quickref.html#citations).

### CODE REVIEW

**Comment by atavory** [comment]
@rschlussel Hi, split and rebased this, as you requested. Could you please review? Many thanks!

File: presto-docs/src/main/sphinx/functions/aggregate.rst [hunk 1]
```diff
@@ -446,4 +441,8 @@ Statistical Aggregate Functions
 
 .. function:: var_samp(x) -> double
 
+--------------------------
+
+.. [BenHaimTomTov2010] Yael Ben-Haim and Elad Tom-Tov, "A streaming parallel decision tree algorithm",
+    J. Machine Learning Research 11 (2010), pp. 849--872.
     Returns the sample variance of all input values.
```
Comment (rschlussel) [inline presto-docs/src/main/sphinx/functions/aggregate.rst]: this line should be above the citation.

Comment (atavory) [reply presto-docs/src/main/sphinx/functions/aggregate.rst]: @rschlussel,

Thanks for your comment! I wasn't sure I got it. The `-----------------` is just an rst directive for a horizontal line, and visually separates the text from the entire citation section (which, in this PR, consists of a single citation). Please LMK if I can clarify anything further.

Comment (rschlussel) [reply presto-docs/src/main/sphinx/functions/aggregate.rst]: ah i see how that wasn't clear.  I meant that line 448 is the description of var_samp(x), so it should be above the citation.

Comment (atavory) [reply presto-docs/src/main/sphinx/functions/aggregate.rst]: Got it. Thanks for spotting this!

**Pull request closed without merging**